frontend/src/views/Login/Login.js

```diff
@@ -12,6 +12,9 @@ import {
   FormLink,
   FormLogIn,
   FormSubmit,
+  RecoverPassword,
+  RecoverPasswordText,
+  RecoverSpan,
```

Comment:
Tutaj bym się trzymał jednej konwencji, więc albo RecoverText albo RecoverPasswordSpan. To drugie chyba lepsze.